typeshed_client/parser.py

```diff
@@ -65,16 +64,11 @@ def parse_ast(
     search_context: SearchContext,
     module_name: ModulePath,
     *,
+    file_path: Path,
     is_init: bool = False,
-    file_path: Optional[Path] = None,
-    is_py_file: bool = False,
 ) -> NameDict:
```

Comment:
this is strictly a breaking change, since `file_path` used to be an optional argument and now it is required, and `parse_ast` is public API exposed by this library. Not sure if that's okay.

There are no uses of this API internally that ever pass `None` to `file_path`, and it simplifies things quite nicely to have a single source of truth for whether a file is a Python file or not, rather than passing the separate `is_py_file` variable around everywhere

Reply:
I think that's fine.